src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -53297,6 +53297,9 @@ static void F(bool b)
             Assert.Equal("void Program.M<Program>(System.Func<Program> t)", model.GetSymbolInfo(invocationNode).Symbol.ToTestDisplayString());
 
             comp.VerifyDiagnostics(
+                // (10,28): warning CS8625: Cannot convert null literal to non-nullable reference type.
+                //                 return new(null);
+                Diagnostic(ErrorCode.WRN_NullAsNonNullable, "null").WithLocation(10, 28),
```

Comment:
This is a duplicated warning. We should postpone visiting args after conversion, but I haven't figured how to do that.